cirq-core/cirq/ops/pauli_string.py

```diff
@@ -1102,19 +1092,13 @@ def _validate_qubit_mapping(
 
 def _try_interpret_as_pauli_string(op: Any):
     """Return a reprepresentation of an operation as a pauli string, if it is possible."""
-    if isinstance(op, gate_operation.GateOperation):
-        gates = {
-            common_gates.XPowGate: pauli_gates.X,
-            common_gates.YPowGate: pauli_gates.Y,
-            common_gates.ZPowGate: pauli_gates.Z,
-        }
-        if (pauli := gates.get(type(op.gate), None)) is not None:
-            exponent = op.gate.exponent  # type: ignore
-            if exponent % 2 == 0:
-                return PauliString()
-            if exponent % 2 == 1:
-                return pauli.on(op.qubits[0])
-    return None
+    if not isinstance(op, raw_types.Operation):
+        return None
+
+    pauli_expansion_op = protocols.pauli_expansion(op, default=None)
+    if pauli_expansion_op is not None and len(pauli_expansion_op) == 1:
```

Comment:
I'm surprised this compiles/lints without a `return None` at the end. Anyway, if you can add that explicitly (or better, invert the condition here to return None if the condition isn't satisfied), and also add a return type `-> PauliString | None` to the function signature, I think it's ready to go.